jupyterhub/values.yaml

```diff
@@ -479,10 +467,7 @@ prePuller:
     podSchedulingWaitDuration: 10
     nodeSelector: {}
     tolerations: []
-    resources:
-      requests:
-        cpu: 0
-        memory: 0
+    resources: {}
```

Comment:
I think I consider this a bugfix actually, the hook-image-awaiter job run's a go program speaking with k8s api-server, it doesn't make sense for this to not request any resources, but it would make sense for it to request a very very low amount of resources during the brief time it lives I think. It should not need to depend on the deployment either.

I think I prefer that we provide defaults resource requests for situations when it doesn't matter if the deployment have 10 or 5000 users or 1 or 100 nodes.